service/lib/agama/software/manager.rb

```diff
@@ -589,6 +589,7 @@ def missing_product_issue
       # @return [Agama::Issue]
       def missing_registration_issue
         Issue.new(_("Product must be registered"),
+          kind:     :missing_registration,
```

Comment:
Interesting that we need similar way in questions. At first it was just string and then we need to add "class" to quickly check what kind it is. :+1: